packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -40,6 +50,9 @@ export interface Theme {
   };
   shadows: ThemeShadowDefinition;
   readonly spacing: Spacing;
+  readonly stroke: {
```

Comment:
I am trying to understand this - why do we need to add the strokes to theme object ( instead of accessing using `globalTokens.stroke.thin `as other endpoints currently). My understanding of theme was it's supposed to be the set of customisations for an end user ( aliases primarily as globals are immutable in design), so why do we want to add this?

Reply:
My understanding is that going forward we don't want to reference global tokens directly and instead go through the theme object, and for tokens we don't want the user to be able to customize, we'll mark them as readonly. This PR is based off the recent work to add iOS spacing tokens. I think @amgleitman or @rurikoaraki have more background behind this decision

Reply:
Spacing is different because it already exists on the Theming object, so it would be weird for it to be haphazardly missing some spacing tokens. I asked @amgleitman to mark them readonly to make clear that they are unable to be messed with, to align with how FluentUI libraries view global tokens.

However, in general I would rather we use global tokens directly to avoid bloating the Theme object.